_rules/sequentially-focusable-element-has-visible-focus-oj04fd.md

```diff
@@ -143,6 +180,113 @@ The [focusable][] element does not have any pixel changing color when it is [foc
 <a class="no-focus-default" href="https://act-rules.github.io/">ACT rules</a>
 ```
 
+#### Failed Example 2
+
+The [focusable][] element does not have any pixel changing color when it is [focused][] because the focus indicator's color matches the background's color, making the outline imperceptible.
+
+```html
+<style>
+	body {background: white;}
+	a:focus {outline-color: white;}
+</style>
+<a href="https://act-rules.github.io/">ACT rules</a>
+```
+
+#### Failed Example 3
+
+The [focusable][] element does not have any pixel changing color when it is [focused][] because the focus indicator's is covered by another element.
```

Comment:
We talked this through in the CG call today. We think this passes the rule, and probably the success criterion too. WCAG does not require different elements to have different focus indicators. There's a comment about this point in the background section:

> WCAG does not require that the focus indicator for each [focusable][] element is unique in appearance. Therefore, this rule can pass even if several focus indicators are identical. Such a situation may nonetheless cause confusion and all examples in this rule avoid it.

We also felt this is enough of an edge case that it isn't worth trying to address.

Reply:
I'm not sure I'm following. Are we referring to point 3? When you run the example, nothing changes in the visual appearance of the page when the element receives focus. Can you please explain a bit more?